tests/foreman/cli/test_host.py

```diff
@@ -926,7 +926,6 @@ def test_positive_list(self):
             self.assertTrue(client.subscribed)
             hosts = Host.list({
                 'organization-id': self.new_org['id'],
-                'environment-id': self.new_lce['id'],
```

Comment:
why we completely remove environment-id from here?

Reply:
As per below output, we don't have environment-id then I thought to use `'puppet-environment-id '` but if you see the test case we are passing lifecycle-environment id to that. isn't that test case is kind wrong? and I'm not sure how is running and passing?     

```
# hammer host list --help | grep environ
 --puppet-environment PUPPET_ENVIRONMENT_NAME  Puppet environment name
 --puppet-environment-id PUPPET_ENVIRONMENT_ID  

```